### ISSUE TITLE
Leere Table in Bereiche Module

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-lectures#149]
Beispiel
https://my2.siteimprove.com/Inspector/764733/A11Y/Page?pageId=113205579708&impmd=1A3574CC7C3B5ABC4A1CA70A1865D7A2&conf=a+aa+aaa+aria&decision=dismissed+falsePositive+none#/sia-r68/failed/aCtBdDBiYWs=;ZWNiMzRhNzM=

https://www.beta.wordpress.rrze.fau.de/plugins/rrze-lectures/beispielsuche-2023w-lecture_name/einzelveranstaltungen-mit-verschiedenen-show-hide-parametern-im-format-tabs/
